src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCorrectIfAndParameter.java

```diff
@@ -114,14 +116,15 @@ void fooMethodWithIf(String stringStringStringStringLooooongString, //indent:6 e
         if (conditionSecond(10000000000.0, new //indent:8 exp:8
             SecondClassLongNam7("Looooooooooooo" //indent:12 exp:12
             + "oooooong").getString(new InputIndentationCorrectIfAndParameter(), //indent:12 exp:12
-            new SecondClassLongNam7("loooooooooong"). //indent:12 exp:12
+            new SecondClassLongNam7("loooooooooong"). //indent:12 exp:14,16 warn
             getInteger(new InputIndentationCorrectIfAndParameter(), "lg")), "l") //indent:12 exp:12
             || conditionThird(2048) || conditionFourth(new //indent:12 exp:12
             SecondClassLongNam7("Looooooooooooooo" //indent:12 exp:12
               + "o").gB(new InputIndentationCorrectIfAndParameter(), false))|| //indent:14 exp:>=12
               conditionFifth(true, new SecondClassLongNam7(getString(2048, "L" //indent:14 exp:>=12
               + "ooooooooooooooooooooooooooooooooooooooooooong")).gB( //indent:14 exp:>=12
-              new InputIndentationCorrectIfAndParameter(),true))||co(false,new //indent:14 exp:>=12
+              new InputIndentationCorrectIfAndParameter(),true))//indent:14 exp:16,18 warn
+                || co(false, new //indent:16 exp:16
```

Comment:
Had to change the pre-written file because it is more than 100 chars per lines due to addition of "warn" comment.